app/controllers/HonestyDeclarationController.scala

```diff
@@ -95,11 +94,10 @@ class HonestyDeclarationController @Inject()(honestDeclarationPage: HonestyDecla
     Future[Result])(implicit request: UserRequest[_]): Future[Result] = {
     (request.answers.getAnswer[String](SessionKeys.reasonableExcuse), request.answers.getAnswer[LocalDate](SessionKeys.dueDateOfPeriod),
       request.answers.getAnswer[LocalDate](SessionKeys.startDateOfPeriod), request.answers.getAnswer[LocalDate](SessionKeys.endDateOfPeriod),
-      request.answers.getAnswer[Boolean](SessionKeys.isObligationAppeal).isDefined) match {
+      request.answers.getAnswer[Boolean](SessionKeys.isFindOutHowToAppeal).isDefined) match {
       case (Some(reasonableExcuse), Some(dueDate), Some(startDate), Some(endDate), isObligation: Boolean) =>
```

Comment:
this param should not be needed anymore (only users with a reasonable excuse can access the honesty declaration page
```suggestion
      ) match {
      case (Some(reasonableExcuse), Some(dueDate), Some(startDate), Some(endDate)) =>
```